### PR TITLE
Implemented getDifference function

### PR DESCRIPTION
**Pull request by Maiia2023**

### CODE REVIEW

File: functions/getDifference/getDifference.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { getDifference } from "./getDifference.js";
```
Comment (Maiia2023) [inline functions/getDifference/getDifference.test.js:1]: Use single quotes

Comment (nagorapro) [inline functions/getDifference/getDifference.test.js:1]: Use single quotes

File: functions/getDifference/getDifference.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/**
+ * @function getDifference
+ * @param {number} a 
+ * @param {number} b 
+ * @returns {number | undefined}
+ */
+
+export const getDifference = (a, b) => {
+  const isValida = typeof a === 'number' &&
```
Comment (nagorapro) [inline functions/getDifference/getDifference.js:9]: Use camelCase style naming

File: functions/getDifference/getDifference.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    a > 0 &&
+    a >= b &&
+    Number.isInteger(a);
+
+  const isValidb = typeof b === 'number' &&
```
Comment (nagorapro) [inline functions/getDifference/getDifference.js:14]: Use camelCase style naming

**Review by nagorapro — COMMENTED** [review]
Fix

**Review by nagorapro — APPROVED** [review]
Done

**Pull request merged**